tests/sources/fixtures/redis/fixture.py

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+#
+# ruff: noqa: T201
+import os
+import random
+
+import redis.asyncio as redis
+
+from tests.commons import WeightedFakeProvider
+
+DATA_SIZE = os.environ.get("DATA_SIZE", "small").lower()
+_NUM_TABLES = {"small": 2, "medium": 4, "large": 6}
+NUM_TABLES = _NUM_TABLES[DATA_SIZE]
+ENDPOINT = "redis://localhost:6379/"
+
+fake_provider = WeightedFakeProvider(weights=[0.65, 0.3, 0.05, 0])
+
+
+async def inject_lines(redis_client, table, start, lines):
+    rows = {}
+    for row_id in range(lines):
+        row_id += start
+        key = f"user_{row_id}"
```

Comment:
What's the purpose of `start` here? Even though you don't use `start`, you can guarantee the uniqueness of the key within this database. Or is it required that the key must be unique across databases?

If it's required, I think you can also code it like
```suggestion
        key = f"user_{row_id + start}"
```

Instead of assigning it to `row_id`.